sites/website/package.json

```diff
@@ -4,15 +4,17 @@
   "private": true,
   "scripts": {
     "clean": "node ../../build/clean.js docs",
-    "prepare-docs": "npm run clean && node src/prepare-docs.js",
-    "dev": "npm run prepare-docs && docusaurus start",
-    "build": "npm run prepare-docs && docusaurus build"
+    "prepare-docs": "yarn clean && node src/prepare-docs.js",
```

Comment:
I'm curious if we have a standard in place for script names.  I've seen hyphens used and colons.  Should we be consistent?  Should it be `prepare:docs` to match `build:homepage` or `prepare-docs` to match `build-homepage`?

Reply:
I think historically, I've always named them with colons to sync with other task runner conventions, [namely Grunt multi-tasks](https://gruntjs.com/creating-tasks#multi-tasks). Some other tools use it but there's no hard rules on naming. Since there's already a convention present, I'll change the task to match.